### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by BadassBison** [BadassBison/Kids-Work#45]
[ ] Logo update
[ ] add gif for quick overview
[ ] add code snippets

**Comment by grantpaulson6** [comment]
Yep agree we need to update readme.  I've got a giphy program that makes it super easy to make gifs so can whip that out.  Before we do however I think we need to complete the site, mostly changing the name, add demo user, and demo user seed data.
